x/wasm/keeper/querier_test.go

```diff
@@ -597,7 +616,8 @@ func TestQueryCodeList(t *testing.T) {
 					Key:    []byte("test"),
 				},
 			},
-			expErr: fmt.Errorf("invalid request, either offset or key is expected, got both"),
+			expPaginationTotal: 0,
```

Comment:
It seems you have not set it in above test functions. So I think you should stay consistent by either setting expPaginationTotal to 0 for "with invalid pagination key" in all test functions, or not setting it at all.
The same problem also occurs on "with empty request", "none"

Reply:
It is as you pointed out. Unify without setting when an error occurs.